image/v1/types.go

```diff
@@ -267,6 +280,14 @@ type TagImportPolicy struct {
 	Insecure bool `json:"insecure,omitempty" protobuf:"varint,1,opt,name=insecure"`
 	// Scheduled indicates to the server that this tag should be periodically checked to ensure it is up to date, and imported
 	Scheduled bool `json:"scheduled,omitempty" protobuf:"varint,2,opt,name=scheduled"`
+	// Platforms indicates which manifest platforms to import. Possible values depend on what is
+	// available under the current tag being imported, and should be given as
+	// '<platform>/<architecture>[/<variant>]'. To import manifests for all available platforms
+	// use the special "*" value. When not specified, only a single manifest is imported
+	// from the manifest list (the manifest list itself is discarded), the architecture and os
+	// are then chosen in the following order of priority: 1. tag annotations; 2. control plane
+	// arch/os; 3. linux/amd64; 4. the first manifest in the list.
+	Platforms []string `json:"platforms,omitempty" protobuf:"bytes,3,rep,name=platforms"`
```

Comment:
so i'm guessing the implementation will populate all the values under the tag being imported, i.e if the image is manifestlisted, all manifests will be imported?

Reply:
the ultimate idea is to allow users to import only a subset of platforms. this might be subject to change however, depending on the engineering cost. to import all manifests from a manifest list the platform should be set to `*`. an empty list of platforms will result in the legacy/current behavior where the manifest list is discarded and only one specific architecture is imported as a single manifest.
does that answer your question?

Reply:
The default behavior won't change (as otherwise it'll break backward compatibility) - it'll import only one image for the control plane arch. If you want to enable the new behavior, you need to provide a list of platforms that you want to use or `*`.

Reply:
Is there a list of valid values for this slice that we can document? How often is that list likely to change?

Reply:
typical values are  `linux/amd64`, `linux/arm64`, `linux/ppc64le`, `linux/s390x` ? not sure if there are many cases where images are built on an OS other than linux.

Reply:
Added some example values, PTAL